problem_builder/platform_dependencies.py

```diff
@@ -29,10 +29,19 @@
 
 # pylint: disable=unused-import
 
+try:
+    from courseware.models import StudentModule
+except Exception:
+    try:
+        from lms.djangoapps.courseware.models import StudentModule
+    except ImportError:
+        # If we are not running within edx-platform
+        # (e.g., we are running problem-builder unit tests).
+        StudentModule = None
```

Comment:
@mudassir-hafeez, could you please add an explanation that we're adding this for Juniper compatibility and what happens if it's not here?

Reply:
Addressed.